src/schemas/intents.json

```diff
@@ -342,6 +345,37 @@
         }
       ]
     },
+    "PreLoadIntent": {
+      "description": "A Firebolt compliant representation of a user intention to pre-load an app.",
+      "title": "PreLoadIntent",
+      "allOf": [
+          {
+              "$ref": "#/definitions/Intent"
+          },
+          {
+              "$ref": "#/definitions/IntentProperties"
+          },
+          {
+              "type": "object",
+              "properties": {
+                  "action": {
+                      "const": "preload"
+                  }
+              },
+              "not": {
+                  "required": [ "data" ]
+              }
+          }
+      ],
+      "examples": [
+          {
+              "action": "preload",
+              "context": {
+                  "source": "voice"
```

Comment:
Minor detail, but "voice" is an odd example of a preload intent source. we have a value of "other" that is probably more appropriate for this use case.